internal/metadatastorage/sqlstore/store.go

```diff
@@ -221,6 +221,127 @@ func (s *Store) Store(ctx context.Context, gitoid string, obj []byte) error {
 	return nil
 }
 
+func (s *Store) storePolicy(ctx context.Context, envelope *dsse.Envelope, gitoid string) error {
```

Comment:
Would it be a terrible idea to suggest storing a Policy as a predicate inside a statement? This could simplify the code and allow us to define multiple subjects of the policy in the future. What do you think @mikhailswift and @kairoaraujo?

Reply:
Although, we'd need to make sure the subjects aren't something that change over time... or if they do, it's within a workflow that makes sense to re-sign a policy. We can also just have any subjects and the flow would be the same until the predicate parserStorer.

Reply:
The idea of storing a policy in a statement has come up before, and it is an appealing option. The only drawback I can see is that it's a drift from where upstream in-toto is moving as far as specifications for the layout/policy go.

Reply:
I'll bring this up at the community call tomorrow. It's probably best to go ahead as is until we get some feedback from the community.